kubernetes/crypto_detector/stats/stats.csv

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do we need this empty file? Not a good idea.

Reply:
Now the file is created and deleted within the script